Header header
float32 xL	#Rotational x vector component
float32 yL      #Rotational y vector component
float32 xR	#Translational x vector component
float32 yR	#Translational y vector component
int32 xButton	#X button state from the joystick
int32 yButton   #Y button state from the joystick
